src/index.js

```diff
@@ -1,14 +1,86 @@
 import './styles/main.scss';
 import showPop from './popup.js';
+import movieImg from './assets/video-player.png';
+
+// disable linters for too many classes
+
+class Likes {
+  static getLikes = async () => {
+    const res = await fetch(
+      'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/w6gyfRjKef7dpeJ8lwcd/likes',
+    );
+    const data = await res.json();
+    return data;
+  };
+
+  static addLikes = async (id) => {
+    const response = await fetch(
+      'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/w6gyfRjKef7dpeJ8lwcd/likes',
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          item_id: id,
+        }),
+      },
+    );
+    const post = await response.text();
+    return post;
+  };
+
+  static postComments = async (id, name, comment) => {
+    const res = await fetch(
+      'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/w6gyfRjKef7dpeJ8lwcd/comments',
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          item_id: id,
+          username: name,
+          comment,
+        }),
+      },
+    );
+    const post = await res.text();
+    return post;
+  };
+
+  static getComments = async (id) => {
```

Comment:
- [ ] It would be better if we create a counter function to keep track of the counters for each page, also we are expected to write a function to test the counter function

Reply:
Function created for total numbers of movies count on home page and total numbers of comments on popup page